omnibus/config/software/chefdk.rb

```diff
@@ -39,17 +39,13 @@
 
 relative_path "chef-dk"
 
-if windows?
-  dependency "ruby-windows"
-  dependency "ruby-windows-devkit"
-else
-  dependency "libffi" if debian?
-  dependency "ruby"
-end
+dependency "ruby"
 
 dependency "rubygems"
 dependency "bundler"
 dependency "appbundler"
+# windows does not have native readline support with compiled ruby
+dependency "rb-readline" if windows?
```

Comment:
reiterating this - it'll also make https://github.com/chef/chef-dk/pull/748 redundant and keep things consistent.